modules/virt-accessing-vmi-ssh.adoc

```diff
@@ -4,51 +4,84 @@
 
 :_content-type: PROCEDURE
 [id="virt-accessing-vmi-ssh_{context}"]
-= Accessing a virtual machine instance via SSH
+= Accessing a virtual machine via SSH by using virtctl
 
-You can use SSH to access a virtual machine (VM) after you expose port 22 on it.
+You can use the `virtctl ssh` command to forward SSH traffic to a virtual machine (VM) by using your local SSH client.
```

Comment:
Leaving out `by using your local SSH client` is enough to prevent false assumptions. Internally the mechanism works differently from 4.12 on but from this example it is not noticeable.

Reply:
Fixing in https://github.com/openshift/openshift-docs/pull/55100